x/bank/internal/types/genesis.go

```diff
@@ -1,18 +1,88 @@
 package types
 
-// GenesisState is the bank state that must be provided at genesis.
+import (
+	"bytes"
+	"encoding/json"
+	"sort"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/bank/exported"
+)
+
+var _ exported.GenesisBalance = (*Balance)(nil)
+
+// GenesisState defines the bank module's genesis state.
 type GenesisState struct {
-	SendEnabled bool `json:"send_enabled" yaml:"send_enabled"`
+	SendEnabled bool      `json:"send_enabled" yaml:"send_enabled"`
+	Balances    []Balance `json:"balances" yaml:"balances"`
```

Comment:
Have we migrated the `gaiad add-genesis-account` command as part of this work?

Reply:
That'll be handled in Gaia once this PR is merged into `master` 👍

Reply:
Just found this out! Thank you @alexanderbez !

Reply:
Correct @AdityaSripal and already done a la https://github.com/cosmos/cosmos-sdk/pull/5589 ;-)